exonum/src/helpers/fabric/maintenance.rs

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![allow(missing_debug_implementations)]
+
+//! This module implements node maintenance actions.
+// spell-checker:ignore exts
+
+use std::path::Path;
+use std::collections::HashMap;
+
+use blockchain::Schema;
+use helpers::config::ConfigFile;
+use storage::{Database, DbOptions, RocksDB};
+use node::NodeConfig;
+use super::internal::{CollectedCommand, Command, Feedback};
+use super::{Argument, CommandName, Context};
+
+const NODE_CONFIG_PATH: &str = "NODE_CONFIG_PATH";
+const DATABASE_PATH: &str = "DATABASE_PATH";
+const MAINTENANCE_ACTION_PATH: &str = "MAINTENANCE_ACTION_PATH";
+
+/// Maintenance command.
```

Comment:
I'm disappointed about current state of `fabric` module docs. I would like to start improving it. What about briefly describing all the arguments?

Reply:
I thought about it too. I think we should first come to agreement on conventions.